### DISCUSSION TITLE
Option for hiding the launcher

### DISCUSSION THREAD

**Discussion by MuhammadBayiz** [rodit/SnapMod#238] [Ideas & Feature Suggestions]
Having the module icon in he launcher it's something uncomfortable and bad for the eyes for me as a user, having an option for hiding it it's necessary especially when you can open the module settings in Lsposed Manager which make it easier to collect all your modules inside lsposed manager

**Comment by RevealedSoulEven** [comment]
Completely useless

**Comment by GoatBan** [comment]
You have the hidden setting on Android, pinch your screen

**Comment by RevealedSoulEven** [comment]
You should forget about it. Coz the settings that you're talking about is only valid for lsposed. What about people using other xposed frameworks? Like edxposed or xposed? So better you hide the app icon if it pains
